rook/dd/dd-image/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
```

Comment:
we can make this a bit more generic and define args so that it can receive the list of devices and other parameters such as the block size, and the number of blocks, as well as oflags

Reply:
an alternative would also be to use python so that it's easier to provide args, etc.

Reply:
can we open an issue for it and do it later for all the scripts ?